deploy/Vagrantfile

```diff
@@ -0,0 +1,91 @@
+# -*- mode: ruby -*-
+# vi: set ft=ruby :
+
+# All Vagrant configuration is done below. The "2" in Vagrant.configure
+# configures the configuration version (we support older styles for
+# backwards compatibility). Please don't change it unless you know what
+# you're doing.
+Vagrant.configure(2) do |config|
+  # The most common configuration options are documented and commented below.
+  # For a complete reference, please see the online documentation at
+  # https://docs.vagrantup.com.
+
+  ### COMMON VARIABLES
+  
+  VAGRANT_ROOT = File.dirname(File.expand_path(__FILE__))
+
+  ### SHARED COMPONENT
+  
+  # GlusterFS File Server 1
+  disk_file_1 = File.join(VAGRANT_ROOT, 'gfs_brick_1.vdi')
+  config.vm.define :file_server_1 do |fs_config|
+    fs_config.vm.box = "chef/centos-6.5"
```

Comment:
No, Vagrant doesn't support backwards compatibility like that, you have to choose which configuration notation to use (hence the "2" in Vagrant.configure()).  The vast majority of people will be on Vagrant 1.1+, which uses version 2 notation (the latest version is 1.7.2).  If we really want to, we could add parallel sections for each VM using version 1, but I don't think that's really needed unless someone raises it as an issue.